src/main/java/ru/hogwarts/school/controller/StudentController.java

```diff
@@ -60,4 +62,22 @@ public ResponseEntity<Collection<Student>> getAll() {
         Collection<Student> students = service.getAll();
         return ResponseEntity.ok(students);
     }
+
+    @GetMapping("/findStudentByAgeBetween")
+    public ResponseEntity<Collection<Student>> findByAgeBetween(@RequestParam(required = true) int max, @RequestParam(required = true) int min) {
+        List<Student> studentToFind = StudentService.findByAgeBetween(min, max);
```

Comment:
Нужно обращаться к объекту, а не к классу. (Все же ООП, да и метод не статический)
```suggestion
        List<Student> studentToFind = service.findByAgeBetween(min, max);
```